### ISSUE TITLE
Types not found

### ISSUE THREAD

**Issue by dominics** [bitttttten/storybook-mock-date-decorator#2]
>  error TS7016: Could not find a declaration file for module 'storybook-mock-date-decorator'. '/some/path/to/app/node_modules/storybook-mock-date-decorator/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/storybook-mock-date-decorator` if it exists or add a new declaration (.d.ts) file containing `declare module 'storybook-mock-date-decorator';`

Maybe it needs a `types` declaration in `package.json`?

**Comment by bitttttten** [comment]
Ohh good point! I'll try and get on this soon. Although if you want an immediate fix, I think you can reference the types somewhere in a .d.ts file in your project right now:

```ts
/// <reference types="storybook-mock-date-decorator/types" />
```

I will keep the issue open until it's fixed or someone provides a fix. Thanks!

**Comment by bitttttten** [comment]
I just published 1.0.1, hoping that this fixes it!
